### PR TITLE
Prefer testing your Ember application with QUnit

### PR DESCRIPTION
**Pull request by seanpdoyle**
https://trello.com/c/VnZehZn1

> Prefer testing your application with [QUnit][ember-test-guides].

[ember-test-guides]: https://guides.emberjs.com/v2.2.0/testing/

### CODE REVIEW

**Comment by MattMSumner** [comment]
:+1: :shipit:

File: best-practices/README.md [hunk 1]
```diff
@@ -308,6 +308,9 @@ Ember
 * Prefer dependency injection through `Ember.inject` over initializers, globals
   on window, or namespaces. ([sample][inject])
 * Prefer sub-routes over maintaining state.
+* Prefer testing your application with [QUnit][ember-test-guides]
```
Comment (joshuaclayton) [inline best-practices/README.md]: For consistency, can we end this in a period?

**Comment by joshuaclayton** [comment]
@seanpdoyle one nitpick, otherwise :tada:

**Comment by drapergeek** [comment]
:+1:

**Comment by joshuaclayton** [comment]
:ship:

**Comment by jferris** [comment]
@seanpdoyle seems like there were no objections to this, so it's good to merge?

**Pull request merged**